### PR TITLE
Ft 49 add trending nfts section

### PR DESCRIPTION
**Pull request by Muhindo-Galien**
# PR Summary

I have added the trending  component.

## Description
the trending component is composed  of:

- image
- currency
- likes
- buy botton
- name of the Nft
- category of the NFT
-

## How Has This Been Tested
it's been good looking

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ x] All new and existing tests passed.

## Screenshots

desk
mobile

### CODE REVIEW

File: lib/web3/service.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ class Web3Service {
     try {
       const web3 = this.web3Instance;
       const hashedData = web3.utils.sha3(data);
-      const signature = await web3.eth.personal.sign(hashedData, address, "");
+      const signature = await web3.eth.personal.sign(hashedData!, address, "");
```
Comment (eliezerbasubi) [inline lib/web3/service.ts:84]: Can we fill the address parameter with the address of the connected user?

**Comment by eliezerbasubi** [comment]
Can we decrease the font size of `Buy Now` button or increase its height to give it some space to breath

File: lib/web3/service.ts (hunk 1 above)
Comment (Muhindo-Galien) [reply lib/web3/service.ts:84]: changes applied

File: lib/web3/service.ts [hunk 2]
```diff
@@ -75,10 +76,12 @@ class Web3Service {
     data: string,
     address: string
   ): Promise<SignResult | null> {
+    let signer: any;
     try {
       const web3 = this.web3Instance;
       const hashedData = web3.utils.sha3(data);
-      const signature = await web3.eth.personal.sign(hashedData, address, "");
+      signer = LocalStorage.getItem("ongama_signer_address");
+      const signature = await web3.eth.personal.sign(hashedData!, signer, "");
```
Comment (eliezerbasubi) [inline lib/web3/service.ts]: Why is the third argument empty?

**Review by eliezerbasubi — APPROVED** [review]
LGTM

File: lib/web3/service.ts (hunk 2 above)
Comment (Muhindo-Galien) [reply lib/web3/service.ts]: i wasn't able to find out what to feel in, so, i'm trying  to reach out to @danmugh  so, he can let me know what to pass as third argument

Comment (eliezerbasubi) [reply lib/web3/service.ts]: Awesome!

Let me know when it's done.

Comment (Muhindo-Galien) [reply lib/web3/service.ts]: okay sir

**Comment by Muhindo-Galien** [comment]
conflicts resolved, kindly check this out @eliezerbasubi @Jonath-z

**Pull request merged**